packages/terra-application-layout/src/menu/_ApplicationMenu.jsx

```diff
@@ -151,14 +163,19 @@ class ApplicationMenu extends React.Component {
     customProps.className);
 
     const isCompact = Helpers.isSizeCompact(layoutConfig.size);
+    const onBlur = utilityConfig && utilityConfig.onBlur ? this.handleBlur : undefined;
```

Comment:
This variable is not needed.  You can add the value be assigned directly on line 177.